### ISSUE TITLE
[BUG]FATAL:  Timed out: Read RPC to 172.25.0.2:7100 timed out after 1.203s

### ISSUE THREAD

**Issue by vtzan** [yugabyte/yugabyte-db#2581]
Dear Team,

I am using 1 node yugabyte 2.0.1 on GCE VM with 1 CPU / 4GB RAM / 60GB SSD
I have create the node using yb-docker-ctl :

```
./yb-docker-ctl create
WARNING: No swap limit support
docker run --name yb-master-n1 --hostname yb-master-n1 --privileged -p 7000:7000 --net yb-net --detach yugabytedb/yugabyte:latest /home/yugabyte/bin/yb-master --replication_factor=1 --master_addresses=yb-master-n1:7100 --rpc_bind_addresses=yb-master-n1:7100 --use_initial_sys_catalog_snapshot=true --fs_data_dirs=/mnt/disk0,/mnt/disk1 --yb_num_shards_per_tserver=2 --v=0
Adding node yb-master-n1
docker run --name yb-tserver-n1 --hostname yb-tserver-n1 --privileged -p 9000:9000 -p 9042:9042 -p 6379:6379 -p 5433:5433 --net yb-net --detach yugabytedb/yugabyte:latest /home/yugabyte/bin/yb-tserver --tserver_master_addrs=yb-master-n1:7100 --rpc_bind_addresses=yb-tserver-n1:9100 --memory_limit_hard_bytes=8589934592 --use_cassandra_authentication=false --start_pgsql_proxy --pgsql_proxy_bind_address yb-tserver-n1:5433 --fs_data_dirs=/mnt/disk0,/mnt/disk1 --yb_num_shards_per_tserver=2 --v=0
Adding node yb-tserver-n1
ID             PID        Type       Node                 URL                       Status          Started At          
ed36040dc3e5   18571      tserver    yb-tserver-n1        http://172.25.0.3:9000    Running         2019-10-11T06:43:18.573083086Z
cfcd7241be29   18404      master     yb-master-n1         http://172.25.0.2:7000    Running         2019-10-11T06:43:17.628405045Z
```

VM SSD test just in case:
```
sync; dd if=/dev/zero of=tempfile0 bs=512k count=1024; sync
1024+0 records in
1024+0 records out
536870912 bytes (537 MB, 512 MiB) copied, 3.15766 s, 170 MB/s

sync; dd if=/dev/zero of=tempfile bs=1M count=1024; sync
1024+0 records in
1024+0 records out
1073741824 bytes (1.1 GB, 1.0 GiB) copied, 9.06574 s, 118 MB/s

sync; dd if=/dev/zero of=tempfile1 bs=2M count=1024; sync
1024+0 records in
1024+0 records out
2147483648 bytes (2.1 GB, 2.0 GiB) copied, 24.1899 s, 88.8 MB/s

```

I am importing postgres schema using the bash script below: 

```
#!/bin/bash

wget https://downloads.yugabyte.com/yugabyte-2.0.1.0-linux.tar.gz
tar xvfz yugabyte-2.0.1.0-linux.tar.gz
cd yugabyte-2.0.1.0/
./bin/post_install.sh
git clone https://github.com/OpenSIPS/opensips.git

DB="opensips_edge_v1";

./bin/ysqlsh -c "create database $DB;"
./bin/ysqlsh -f opensips/scripts/postgres/standard-create.sql $DB
./bin/ysqlsh -f opensips/scripts/postgres/acc-create.sql $DB
./bin/ysqlsh -f opensips/scripts/postgres/alias_db-create.sql $DB
./bin/ysqlsh -f opensips/scripts/postgres/auth_db-create.sql $DB
./bin/ysqlsh -f opensips/scripts/postgres/avpops-create.sql $DB
./bin/ysqlsh -f opensips/scripts/postgres/b2b-create.sql $DB
./bin/ysqlsh -f opensips/scripts/postgres/b2b_sca-create.sql $DB
./bin/ysqlsh -f opensips/scripts/postgres/cachedb.sql $DB-create.sql $DB
./bin/ysqlsh -f opensips/scripts/postgres/call_center-create.sql $DB
./bin/ysqlsh -f opensips/scripts/postgres/carrierroute-create.sql $DB
./bin/ysqlsh -f opensips/scripts/postgres/closeddial-create.sql $DB
./bin/ysqlsh -f opensips/scripts/postgres/clusterer-create.sql $DB
./bin/ysqlsh -f opensips/scripts/postgres/cpl-create.sql $DB
./bin/ysqlsh -f opensips/scripts/postgres/dialog-create.sql $DB
./bin/ysqlsh -f opensips/scripts/postgres/dialplan-create.sql $DB
./bin/ysqlsh -f opensips/scripts/postgres/dispatcher-create.sql $DB
./bin/ysqlsh -f opensips/scripts/postgres/domain-create.sql $DB
./bin/ysqlsh -f opensips/scripts/postgres/domainpolicy-create.sql $DB
./bin/ysqlsh -f opensips/scripts/postgres/drouting-create.sql $DB
./bin/ysqlsh -f opensips/scripts/postgres/emergency-create.sql $DB
./bin/ysqlsh -f opensips/scripts/postgres/fraud_detection-create.sql $DB
./bin/ysqlsh -f opensips/scripts/postgres/freeswitch_scripting-create.sql $DB
./bin/ysqlsh -f opensips/scripts/postgres/group-create.sql $DB
./bin/ysqlsh -f opensips/scripts/postgres/imc-create.sql $DB
./bin/ysqlsh -f opensips/scripts/postgres/load_balancer-create.sql $DB
./bin/ysqlsh -f opensips/scripts/postgres/msilo-create.sql $DB
./bin/ysqlsh -f opensips/scripts/postgres/permissions-create.sql $DB
./bin/ysqlsh -f opensips/scripts/postgres/presence-create.sql $DB
./bin/ysqlsh -f opensips/scripts/postgres/registrant-create.sql $DB
./bin/ysqlsh -f opensips/scripts/postgres/rls-create.sql $DB
./bin/ysqlsh -f opensips/scripts/postgres/rtpengine-create.sql $DB
./bin/ysqlsh -f opensips/scripts/postgres/smpp-create.sql $DB
./bin/ysqlsh -f opensips/scripts/postgres/speeddial-create.sql $DB
./bin/ysqlsh -f opensips/scripts/postgres/tls_mgm-create.sql $DB
./bin/ysqlsh -f opensips/scripts/postgres/tracer-create.sql $DB
./bin/ysqlsh -f opensips/scripts/postgres/userblacklist-create.sql $DB
./bin/ysqlsh -f opensips/scripts/postgres/usrloc-create.sql $DB
```

The import takes about 3m2.973s to finish. Isn't it too much? or this is just how it works?
Finaly the problem is those logs regarding timeouts. Is this normal?

```
...........
W1011 06:51:37.476565   182 yb_rpc.cc:359] Call yb.tserver.TabletServerService.Read 172.25.0.3:53334 => 172.25.0.2:7100 (request call id 839) took 1213ms (client timeout 1202ms).
**W1011 06:51:37.476565   182 yb_rpc.cc:359] Call yb.tserver.TabletServerService.Read 172.25.0.3:53334 => 172.25.0.2:7100 (request call id 839) took 1213ms (client timeout 1202ms).**
==> postgresql-2019-10-11_064321.log <==
I1011 06:51:36.945430  3337 table_creator.cc:256] Created table opensips_edge_v1.location of type PGSQL_TABLE_TYPE
W1011 06:51:37.467459  2379 tablet_rpc.cc:327] Timed out (yb/rpc/outbound_call.cc:510): Failed Read(tablet: 00000000000000000000000000000000, num_ops: 1, num_attempts: 47, txn: 00000000-0000-0000-0000-000000000000) to tablet 00000000000000000000000000000000 on tablet server { uuid: ded64f2140d34e8db40334aa95743965 private: [host: "yb-master-n1" port: 7100] cloud_info: placement_cloud: "cloud1" placement_region: "datacenter1" placement_zone: "rack1" after 47 attempt(s): Read RPC to 172.25.0.2:7100 timed out after 1.203s
**2019-10-11 06:51:37.467 UTC [2368] FATAL:  Timed out: Read RPC to 172.25.0.2:7100 timed out after 1.203s**

```
Here is the status shown in tablet-servers

![tablet-servers-status](https://user-images.githubusercontent.com/1777931/66632001-296aae80-ec10-11e9-85e7-537a6d617341.png)

I am also attaching logs from both docker containers after import script.

[logs.zip](https://github.com/yugabyte/yugabyte-db/files/3716117/logs.zip)

thank you in advance
Vasilios Tzanoudakis

**Comment by ddorian** [comment]
@vtzan I will try to reproduce and get back to you

**Comment by ddorian** [comment]
@vtzan can you please list your tables on the database with `\d` in `ysqlsh` ? And attach it.
Cause I executed the script and I'm seeing different numbers compared to your screenshots.

**Comment by vtzan** [comment]
Dear @ddorian,

Thank you for your reply.
I am sending you the data you requested.


```
yugabyte=# \c opensips_edge_v1
You are now connected to database "opensips_edge_v1" as user "yugabyte".
opensips_edge_v1=# \d
                        List of relations
 Schema |               Name                |   Type   |  Owner   
--------+-----------------------------------+----------+----------
 public | acc                               | table    | yugabyte
 public | acc_id_seq                        | sequence | yugabyte
 public | active_watchers                   | table    | yugabyte
 public | active_watchers_id_seq            | sequence | yugabyte
 public | address                           | table    | yugabyte
 public | address_id_seq                    | sequence | yugabyte
 public | b2b_entities                      | table    | yugabyte
 public | b2b_entities_id_seq               | sequence | yugabyte
 public | b2b_logic                         | table    | yugabyte
 public | b2b_logic_id_seq                  | sequence | yugabyte
 public | b2b_sca                           | table    | yugabyte
 public | b2b_sca_id_seq                    | sequence | yugabyte
 public | carrierfailureroute               | table    | yugabyte
 public | carrierfailureroute_id_seq        | sequence | yugabyte
 public | carrierroute                      | table    | yugabyte
 public | carrierroute_id_seq               | sequence | yugabyte
 public | cc_agents                         | table    | yugabyte
 public | cc_agents_id_seq                  | sequence | yugabyte
 public | cc_calls                          | table    | yugabyte
 public | cc_calls_id_seq                   | sequence | yugabyte
 public | cc_cdrs                           | table    | yugabyte
 public | cc_cdrs_id_seq                    | sequence | yugabyte
 public | cc_flows                          | table    | yugabyte
 public | cc_flows_id_seq                   | sequence | yugabyte
 public | closeddial                        | table    | yugabyte
 public | closeddial_id_seq                 | sequence | yugabyte
 public | clusterer                         | table    | yugabyte
 public | clusterer_id_seq                  | sequence | yugabyte
 public | cpl                               | table    | yugabyte
 public | cpl_id_seq                        | sequence | yugabyte
 public | dbaliases                         | table    | yugabyte
 public | dbaliases_id_seq                  | sequence | yugabyte
 public | dialog                            | table    | yugabyte
 public | dialplan                          | table    | yugabyte
 public | dialplan_id_seq                   | sequence | yugabyte
 public | dispatcher                        | table    | yugabyte
 public | dispatcher_id_seq                 | sequence | yugabyte
 public | domain                            | table    | yugabyte
 public | domain_id_seq                     | sequence | yugabyte
 public | domainpolicy                      | table    | yugabyte
 public | domainpolicy_id_seq               | sequence | yugabyte
 public | dr_carriers                       | table    | yugabyte
 public | dr_carriers_id_seq                | sequence | yugabyte
 public | dr_gateways                       | table    | yugabyte
 public | dr_gateways_id_seq                | sequence | yugabyte
 public | dr_groups                         | table    | yugabyte
 public | dr_groups_id_seq                  | sequence | yugabyte
 public | dr_partitions                     | table    | yugabyte
 public | dr_partitions_id_seq              | sequence | yugabyte
 public | dr_rules                          | table    | yugabyte
 public | dr_rules_ruleid_seq               | sequence | yugabyte
 public | emergency_report                  | table    | yugabyte
 public | emergency_report_id_seq           | sequence | yugabyte
 public | emergency_routing                 | table    | yugabyte
 public | emergency_routing_id_seq          | sequence | yugabyte
 public | emergency_service_provider        | table    | yugabyte
 public | emergency_service_provider_id_seq | sequence | yugabyte
 public | fraud_detection                   | table    | yugabyte
 public | fraud_detection_ruleid_seq        | sequence | yugabyte
 public | freeswitch                        | table    | yugabyte
 public | freeswitch_id_seq                 | sequence | yugabyte
 public | globalblacklist                   | table    | yugabyte
 public | globalblacklist_id_seq            | sequence | yugabyte
 public | grp                               | table    | yugabyte
 public | grp_id_seq                        | sequence | yugabyte
 public | imc_members                       | table    | yugabyte
 public | imc_members_id_seq                | sequence | yugabyte
 public | imc_rooms                         | table    | yugabyte
 public | imc_rooms_id_seq                  | sequence | yugabyte
 public | load_balancer                     | table    | yugabyte
 public | load_balancer_id_seq              | sequence | yugabyte
 public | location                          | table    | yugabyte
 public | location_contact_id_seq           | sequence | yugabyte
 public | missed_calls                      | table    | yugabyte
 public | missed_calls_id_seq               | sequence | yugabyte
 public | presentity                        | table    | yugabyte
 public | presentity_id_seq                 | sequence | yugabyte
 public | pua                               | table    | yugabyte
 public | pua_id_seq                        | sequence | yugabyte
 public | re_grp                            | table    | yugabyte
 public | re_grp_id_seq                     | sequence | yugabyte
 public | registrant                        | table    | yugabyte
 public | registrant_id_seq                 | sequence | yugabyte
 public | rls_presentity                    | table    | yugabyte
 public | rls_presentity_id_seq             | sequence | yugabyte
 public | rls_watchers                      | table    | yugabyte
 public | rls_watchers_id_seq               | sequence | yugabyte
 public | route_tree                        | table    | yugabyte
 public | route_tree_id_seq                 | sequence | yugabyte
 public | rtpengine                         | table    | yugabyte
 public | rtpengine_id_seq                  | sequence | yugabyte
 public | silo                              | table    | yugabyte
 public | silo_id_seq                       | sequence | yugabyte
 public | sip_trace                         | table    | yugabyte
 public | sip_trace_id_seq                  | sequence | yugabyte
 public | smpp                              | table    | yugabyte
 public | smpp_id_seq                       | sequence | yugabyte
 public | speed_dial                        | table    | yugabyte
 public | speed_dial_id_seq                 | sequence | yugabyte
 public | subscriber                        | table    | yugabyte
 public | subscriber_id_seq                 | sequence | yugabyte
 public | tls_mgm                           | table    | yugabyte
 public | tls_mgm_id_seq                    | sequence | yugabyte
 public | uri                               | table    | yugabyte
 public | uri_id_seq                        | sequence | yugabyte
 public | userblacklist                     | table    | yugabyte
 public | userblacklist_id_seq              | sequence | yugabyte
 public | usr_preferences                   | table    | yugabyte
 public | usr_preferences_id_seq            | sequence | yugabyte
 public | version                           | table    | yugabyte
 public | watchers                          | table    | yugabyte
 public | watchers_id_seq                   | sequence | yugabyte
 public | xcap                              | table    | yugabyte
 public | xcap_id_seq                       | sequence | yugabyte
(114 rows)
```

Let me know if you need any further information on this case.

thank you in advance
Vasilios Tzanoudakis

**Comment by vtzan** [comment]
After 
1)fixing the system limits
```
$ ulimit -a
core file size          (blocks, -c) unlimited
data seg size           (kbytes, -d) unlimited
scheduling priority             (-e) 0
file size               (blocks, -f) unlimited
pending signals                 (-i) 119934
max locked memory       (kbytes, -l) 64
max memory size         (kbytes, -m) unlimited
open files                      (-n) 65535
pipe size            (512 bytes, -p) 8
POSIX message queues     (bytes, -q) 819200
real-time priority              (-r) 0
stack size              (kbytes, -s) 8192
cpu time               (seconds, -t) unlimited
max user processes              (-u) 12000
virtual memory          (kbytes, -v) unlimited
file locks                      (-x) unlimited
```
2)recreating a single node cluster using yb-ctl
```
$ ./bin/yb-ctl status
----------------------------------------------------------------------------------------------------
| Node Count: 1 | Replication Factor: 1                                                            |
----------------------------------------------------------------------------------------------------
| JDBC                : jdbc:postgresql://127.0.0.1:5433/postgres                                  |
| YSQL Shell          : bin/ysqlsh                                                                 |
| YCQL Shell          : bin/cqlsh                                                                  |
| YEDIS Shell         : bin/redis-cli                                                              |
| Web UI              : http://127.0.0.1:7000/                                                     |
| Cluster Data        : /home/vtzan/yugabyte-data                                                  |
----------------------------------------------------------------------------------------------------
----------------------------------------------------------------------------------------------------
| Node 1: yb-tserver (pid 4386), yb-master (pid 4383)                                              |
----------------------------------------------------------------------------------------------------
| JDBC                : jdbc:postgresql://127.0.0.1:5433/postgres                                  |
| YSQL Shell          : bin/ysqlsh                                                                 |
| YCQL Shell          : bin/cqlsh                                                                  |
| YEDIS Shell         : bin/redis-cli                                                              |
| data-dir[0]         : /home/vtzan/yugabyte-data/node-1/disk-1/yb-data                            |
| yb-tserver Logs     : /home/vtzan/yugabyte-data/node-1/disk-1/yb-data/tserver/logs               |
| yb-master Logs      : /home/vtzan/yugabyte-data/node-1/disk-1/yb-data/master/logs                |
----------------------------------------------------------------------------------------------------

```
I show this error only once during the schema import procecure:

```
$ tail -f /home/vtzan/yugabyte-data/node-1/disk-1/yb-data/master/logs/yb-master.WARNING 
    @     0x7f857d3b19b5  yb::master::ScopedLeaderSharedLock::Unlock()
    @     0x7f857d3e2785  yb::master::MasterServiceImpl::CreateNamespace()
    @     0x7f8577d00079  yb::master::MasterServiceIf::Handle()
    @     0x7f8575eac0d1  yb::rpc::ServicePoolImpl::Handle()
    @     0x7f8575e58764  yb::rpc::InboundCall::InboundCallTask::Run()
    @     0x7f8575eb7c58  yb::rpc::(anonymous namespace)::Worker::Execute()
    @     0x7f85744bfa19  yb::Thread::SuperviseThread()
    @     0x7f856fec3694  start_thread
    @     0x7f856f60041d  __clone
    @              (nil)  (unknown)

```

thank you in advance
Vasilios Tzanoudakis

**Comment by ddorian** [comment]
@vtzan is continuing testing. he was using previously an old `yb-docker-ctl` script that created too many tablets.

**Comment by vtzan** [comment]
Dear @ddorian,

The purpose of this issue was : 
`FATAL: Timed out: Read RPC to 172.25.0.2:7100 timed out after 1.203s 
`
And this is solved now with new docker script and by fixing the system limits.

The only problem now is those errors while importing the script.
```
 @     0x7f857d3b19b5  yb::master::ScopedLeaderSharedLock::Unlock()
    @     0x7f857d3e2785  yb::master::MasterServiceImpl::CreateNamespace()
    @     0x7f8577d00079  yb::master::MasterServiceIf::Handle()
    @     0x7f8575eac0d1  yb::rpc::ServicePoolImpl::Handle()
    @     0x7f8575e58764  yb::rpc::InboundCall::InboundCallTask::Run()
    @     0x7f8575eb7c58  yb::rpc::(anonymous namespace)::Worker::Execute()
    @     0x7f85744bfa19  yb::Thread::SuperviseThread()
    @     0x7f856fec3694  start_thread
    @     0x7f856f60041d  __clone
    @              (nil)  (unknown)
```
do you want to open a new issue for that?

thank you in advance
Vasilios Tzanoudakis

**Comment by vtzan** [comment]
Dear Team,

I am closing this issue as it is solved and continuing the latest error on this issue : https://github.com/yugabyte/yugabyte-db/issues/2633

thank you in advance
Vasilios Tzanoudakis
